src/components/Display.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Kindly do not disable linters as they help us write better code

Reply:
@Ghiftee and even if I install propTypes and specify prototypes for components it will work.
![image](https://user-images.githubusercontent.com/73453971/227730138-7c749385-ecc5-4356-ac99-6e267ba6605f.png)
I am waiting for your reply

Reply:
Kindly try out the solution above @BahirHakimy and ensure that you are importing PropTypes correctly. At the moment, you are importing it from ```prop types``` but you should be importing from ```prop-types```

Reply:
@BahirHakimy, I have tried out the solution I provided on my system and the error is gone, I understand that you are under pressure due to time constraints but please carefully go through it again and try it out 👍🏽

Reply:
@Ghiftee 
Thank you very much but now I get this error
![image](https://user-images.githubusercontent.com/73453971/227731543-90b0a8f5-c8b6-44ff-97b2-86a878db0fb6.png)

Reply:
You need to set the default values like this @BahirHakimy 

Display.defaultProps = {
text: '0',
styles: {},
};

Reply:
@Ghiftee 
Thank you very much, it's solved now
<img src="https://media4.giphy.com/media/3oEdva9BUHPIs2SkGk/giphy.gif"/>

Reply:
You're welcome @BahirHakimy